src/lib/leagues/mediumTasks.ts

```diff
@@ -906,5 +909,84 @@ export const mediumTasks: Task[] = [
 		has: async ({ stashUnits }) => {
 			return stashUnits.filter(i => i.tier.tier === 'Elite').every(i => i.isFull && Boolean(i.builtUnit));
 		}
+	},
+	{
+		id: 1128,
+		name: 'Sacrifice 5 of each bird egg',
```

Comment:
Would probably change this to "Offered" because you use the /offer command not /sacrifice